### PR TITLE
Adjust bio_tools description to the real features

### PR DESCRIPTION
**Pull request by l29ah**
#### Summary
```SUMMARY: Content "Adjust bio_tools description to the real features"```

#### Purpose of change
Fix the disrepancy.

#### Additional context
The data comes from the "toolset" item in data/json/items/fake.json and recipes utilizing it.

### CODE REVIEW

**Comment by KITbutler** [comment]
Typo: "mplanted" instead of "implanted" in both files.

File: data/json/bionics.json [hunk 1]
```diff
@@ -1008,7 +1008,7 @@
     "id": "bio_tools",
     "type": "bionic",
     "name": "Integrated Toolset",
-    "description": "Surgically implanted in your hands and fingers is a complete tool set - screwdriver, hammer, wrench, and heating elements.  You can use this in place of many tools when crafting.",
+    "description": "Surgically mplanted in your hands and fingers is a complete tool set - screwdriver, hammer, wrench, hacksaw, wood saw, boltcutters, drill and heating elements.  You can use this in place of many tools when crafting.",
```
Comment (KorGgenT) [inline data/json/bionics.json]: "Surgically implanted"
Is correct. you removed the i.

File: data/json/items/bionics.json [hunk 2]
```diff
@@ -794,7 +794,7 @@
     "copy-from": "bionic_general",
     "type": "BIONIC_ITEM",
     "name": "Integrated Toolset CBM",
-    "description": "Implanted in your hands and fingers is a complete tool set - screwdriver, hammer, wrench, and heating elements.  You can use this in place of many tools when crafting.",
+    "description": "Surgically mplanted in your hands and fingers is a complete tool set - screwdriver, hammer, wrench, hacksaw, wood saw, boltcutters, drill and heating elements.  You can use this in place of many tools when crafting.",
```
Comment (KorGgenT) [inline data/json/items/bionics.json]: Surgically implanted

**Comment by l29ah** [comment]
On Sun, Jan 27, 2019 at 04:03:28PM -0800, KITbutler wrote:
> Typo: "mplanted" instead of "implanted" in both files.

Fixed.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by ZhilkinSerg** [comment]
Jenkins rebuild.

**Comment by AskaHope** [comment]
I think the bio_tools also have a crowbar.

**Comment by l29ah** [comment]
No, it's just a hammer now is also a weak crowbar.

**Pull request merged**